MIGRATION.md

```diff
@@ -576,6 +578,10 @@ If you were using `viteFinal` in 6.5 to simply merge in your project's standard
 
 Previously, Storybook's Vite builder placed cache files in node_modules/.vite-storybook. However, it's more common for tools to place cached files into `node_modules/.cache`, and putting them there makes it quick and easy to clear the cache for multiple tools at once. We don't expect this change will cause any problems, but it's something that users of Storybook Vite projects should know about. It can be configured by setting `cacheDir` in `viteFinal` within `.storybook/main.js` [Storybook Vite configuration docs](https://storybook.js.org/docs/react/builders/vite#configuration)).
 
+#### SvelteKit needs the `@storybook/sveltekit` framework
+
+SvelteKit projects need to use the `@storybook/sveltekit` framework in the `main.js` file. Previously it was enough to just setup Storybook with Svelte+Vite, but that is no longer the case.
```

Comment:
```suggestion
SvelteKit projects should use the `@storybook/sveltekit` framework in the `main.js` file. Previously it was enough to just setup Storybook with Svelte+Vite, but that is no longer the case.

\`\`\`js
export default {
  framework: '@storybook/sveltekit',
};
\`\`\`
```

Reply:
I'm okay with this suggestion, but you should know that Storybook is unusable if SvelteKit isn't handled explicitly like the framework does. Do you still think it should be changed to "should"?

Reply:
Ok, then "need to" or "must" WFM